lib/src/provider.dart

```diff
@@ -238,7 +238,7 @@ class StatefulProvider<T> extends StatefulWidget
   /// parameter.
   ///
   /// This function is useful when the provided value has custome disposal
-  /// behavior. For example a [Stream] which needs to be closed explicitly.
+  /// behavior. For example a stream which needs to be closed explicitly.
```

Comment:
I think we should change `stream` to `sink`

Reply:
done